scripts/pam-test.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+set -e
+
+# load common functions
+# shellcheck disable=SC1091
+. ./bash_lib.sh
+
+bb_print_env
+
+if ! which mysql ; then
```

Comment:
Are you sure that this is the best way? This would make sure that we have a mysql client, not a server, there might be better and more exhaustive option...

Also, and this is a generic comment, we should use mariadb instead of mysql, I believe that we don't test version (too old) that do not have at least symlink so that mariadb commands would work.

Reply:
This step is in the continuation of the `Install step.`
So I think `which mysql` could be easily removed if we only run the PAM tests step if the installation was `successful`, and that makes perfect sense.

What do you say?